Preprocessing/tst_preprocessing.py

```diff
@@ -0,0 +1,53 @@
+"""
+According to ./README.md
+
+- Source is `/Data/raw`
+  - Folder is named `TST_ResearcherName_YYY-MM_Type` e.g. `TST_Srinivasan_2021-10_QS`
+  - contains following files
+    - `TST_YYYY-MM_Type_metadata.xls`
+      e.g. `TST_2021-10_QS_metadata.xls` (could also be `.xlsx`)
+    - `TST_YYYY-MM_Type_###.csv`
+      e.g. `TST_2021-10_QS_001.csv`, `TST_2021-10_QS_002.csv`, ...
+- Processing `tst_preprocessing.py` does
+  - Parse the raw files
+  - cleanup what can be done automatically
+    - whitespace + unexpected return characters in columns
+    - remove unexpected columns
+  - Validation
+    - Warning messages when not conforming to convention (non-blocking)
```

Comment:
- Information messages also no? (from the test logs I saw)

Reply:
Well ... information messages happen from other part of the script, also.
But I can add it here if considered useful.